libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -596,53 +596,93 @@ export class WorkspaceInstallManager extends BaseInstallManager {
         }
       }
 
-      // Run "npm install" in the common folder
-      const installArgs: string[] = ['install'];
-      this.pushConfigurationArgsForSplitWorkspace(installArgs, this.options);
+      const doInstalInternalSplitWorkspaceAsync = async (): Promise<void> => {
+        // Run "npm install" in the common folder
+        const installArgs: string[] = ['install', '--color=always'];
```

Comment:
Why did you add `--color=always` here? What if Rush is invoked in a context that does not support ANSI color escapes?

Reply:
I also added in the above install. The comments are like this:
```
// To ensure that the output is always colored, set the option "--color=always", even when it's piped.
 // Without this argument, certain text that should be colored (such as red) will appear white.
```

Without this argument, the **piped** pnpm output would default to have no color (https://pnpm.io/npmrc#no-color). In other words, without this argument, all the pnpm output will be white (after adding our `PipeThrough` stream). cmiiw. 


> What if Rush is invoked in a context that does not support ANSI color escapes?

What would be the result of this? I assume the text would just be white, which should be fine.

Reply:
Fixed